test/mir.spec.js

```diff
@@ -3,36 +3,40 @@
 const { describe, it } = require('mocha');
```

Comment:
You might want to remove the comment at the top of this file since you added markdown-it as a dev dependency.

Reply:
good catch. i removed it